src/parser/ParseException.h

```diff
@@ -1,33 +1,49 @@
 // Copyright 2014, University of Freiburg,
 // Chair of Algorithms and Data Structures.
-// Author: Björn Buchhold (buchhold@informatik.uni-freiburg.de)
+// Authors:
+//   2014-2017 Björn Buchhold (buchhold@informatik.uni-freiburg.de)
+//   2022      Julian Mundhahs (mundhahj@tf.informatik.uni-freiburg.de)
 #pragma once
 
 #include <exception>
 #include <string>
+#include <string_view>
 
+// Contains Metadata (position of the faulty clause) about a ParseException that
+// occurred.
 struct ExceptionMetadata {
-  std::string query_;
-  size_t startIndex = 0;
-  size_t stopIndex = 0;
+  // Query that the exception occurred in. This is currently needed because the
+  // parse doesn't parse the whole query in one piece. It therefore can only
+  // determine the position in the current (sub-)query.
+  std::string_view query_;
```

Comment:
```suggestion
  std::string_view query_{};
```
The you can write `{}` instead of `{{}}` when default-constructing without triggering the clang warning.